draft-ietf-dprive-xfr-over-tls.md

```diff
@@ -364,37 +524,128 @@ The figure below provides an outline of the IXoT mechanism including NOTIFYs.
 [Figure 4: IXoT mechanism]
 (https://github.com/hanzhang0116/hzpa-dprive-xfr-over-tls/blob/02_updates/02-draft-svg/IXoT_mechanism_1.svg)
 
-The connection for IXFR-over-TLS SHOULD be established using port 853, as
+The connection for IXFR-over-TLS (IXoT) SHOULD be established using port 853, as
 specified in [@!RFC7858], unless there is mutual agreement between the secondary
-and primary to use a port other than port 853 for XFR-over-TLS.
+and primary to use a port other than port 853 for IXoT.
+
+### Coverage and relationship to RFC1995
 
 [@RFC1995] says nothing with respect to optimizing IXFRs over TCP or re-using
-already open TCP connections to perform IXFRs or other queries. We provide
-guidance here that aligns with the guidance in [@RFC5936] for AXFR and with
+already open TCP connections to perform IXFRs or other queries. We provide new
+guidance here specific to IXoT that aligns with the guidance in [@RFC5936] for AXFR, that in section (#axot-mechanism) for AXoT, and with
 that for performant TCP/TLS usage in [@RFC7766] and [@RFC7858].
 
-An IXoT client MAY use an already opened TLS connection to send a IXFR request.
-Using an existing open connection is RECOMMENDED over opening a new connection.
-(Non-IXoT session traffic can also use an open connection.)
-
-An IXoT client MAY use an already open TLS connection to send an SOA query.
-Using an existing open connection is RECOMMENDED over opening a new connection.
-
-An IXoT server MUST be able to handle multiple IXoT requests on a single TLS
-connection, as well as to handle other query/response transactions over it.
-
-An IXoT client MAY keep an existing TLS session open in the expectation it is
-likely to need to perform an IXFR in the near future. The client may use the
-frequency of recent IXFRs to calculate an average update rate and then use
-EDNS0 Keepalive to request an appropriate timeout from the server (if the
-server supports EDNS0 Keepalive). If the server does not support EDNS0
-Keepalive the client MAY keep the connection open for a few seconds ([@RFC7766]
-recommends that servers use timeouts of at least a few seconds).
-
-An IXoT client MAY pipeline IXFR requests for different zones on a single TLS
-connection. AN IXoT server MAY respond to those requests out of order.
-
-QUESTION: Since this is a new specification should there be a requirement that IXoT servers are RECOMMENDED to condense responses as described in Section 6 of [RFC1995]. [RFC1995] document says this is optional and MAY be done but it can significantly reduce the size of responses and may have implications for padding?
+Where any behavior is not explicitly described here, the behviour specified in
+[@RFC1995] MUST be followed. Any behaviour specified here takes precedence for
+AXoT implementations over that in [@RFC1995].
+
+### IXoT connection and message handling
```

Comment:
I do agree they are very similar but I wanted to keep them separate because the do relate to earlier specifications of IAXFR and IXFR in different ways and I suspect we might have to add some additional statements around handling AXFR sessions that are aborted..... I will try to reference the AXoT text from the IXoT section where possible though. I just thought of an AXoT specific issue with EDNS Keepalive that needs to be mentioned though